Organization/Organization/RequestManager.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import Alamofire
+
+@objc class RequestManager: NSObject
+{
+    static func fetchOrganizations(closure: @escaping ([String : AnyObject]) -> Void)
+    {
+        Alamofire.request("http://gitlab.faifly.com/ios-general/ios-tutorial/uploads/ffffec34fa4a727922bb1e09cc24b21d/tutorial.json").responseJSON{ response in
+                let result = try? JSONSerialization.jsonObject(with: response.data!, options: [])
```

Comment:
You already have serialized JSON in `response.value` (or something like that), no need to serialize data again.